ambry-api/src/test/java/com.github.ambry/account/InMemAccountService.java

```diff
@@ -164,7 +164,10 @@ private Container getRandomContainer(short accountId) {
     boolean refContainerCaching = TestUtils.RANDOM.nextBoolean();
     boolean refContainerMediaScanDisabled = TestUtils.RANDOM.nextBoolean();
     return new ContainerBuilder(refContainerId, refContainerName, refContainerStatus, refContainerDescription,
-        refContainerEncryption, refContainerPreviousEncryption, refContainerCaching, refContainerMediaScanDisabled,
-        accountId).build();
+        accountId).setEncrypted(refContainerEncryption)
+        .setPreviouslyEncrypted(refContainerPreviousEncryption)
+        .setCacheable(refContainerCaching)
+        .setMediaScanDisabled(refContainerMediaScanDisabled)
+        .build();
```

Comment:
we don't set replicationPolicy for this random container?

Reply:
No - it's hard to since the policy will eventually have to match something in the clustermap. It will not be possible to set it without some additional input. When I make the change to use this policy to pick a partition in the `Router`, I will see what can be done